src/makeCalculator.js

```diff
@@ -1,10 +1,58 @@
 'use strict';
 
-/**
- * @return {object}
- */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(callback, number) {
+      this.result = callback(this.result, number);
+
+      return this;
+    },
+
+    add(operand, number) {
+      const result = operand + number;
+
+      return result;
+    },
+
+    subtract(operand, number) {
+      const result = operand - number;
+
+      return result;
```

Comment:
The `subtract` method should use `this.result` instead of `operand`. The method should update `this.result` with the difference between `this.result` and `number`.